modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ViewAllSectionDisplayContext.java

```diff
@@ -51,8 +51,12 @@ public ViewAllSectionDisplayContext(
 	public List<DropdownItem> getBulkActionDropdownItems() {
 		return ListUtil.fromArray(
 			new FDSActionDropdownItem(
-				"#", "trash", "remove",
-				LanguageUtil.get(httpServletRequest, "remove"), null, null,
+				"#", "trash", "delete",
```

Comment:
You can move the `delete` action up to the common `BaseSectionDisplayContext.java` in the general `getBulkActionDropdownItems` and invoke the `super.getBulkActionDropdownItems()` from every single local `getBulkActionDropdownItems` in the peripheral `*DisplayContext.java`